R/create.R

```diff
@@ -30,20 +30,23 @@
 create_package <- function(path,
                            fields = NULL,
                            rstudio = rstudioapi::isAvailable(),
+                           stop_for_name = TRUE,
                            open = interactive()) {
   path <- user_path_prep(path)
   check_path_is_directory(path_dir(path))
 
   name <- path_file(path)
-  check_package_name(name)
+  if (stop_for_name) {
```

Comment:
Do we need this condition? Since we pass `stop_for_name` along, seems like we could execute `check_package_name()` unconditionally.

Reply:
I think the package name needs to be checked before the directory is created to avoid creating an invalid directory and then erroring out, but `use_description()` needs to be called after the directory is created.  I eliminating the warning (below), I've simplified the check below, so still need the conditional here.